src/inc/quic_datapath.h

```diff
@@ -224,7 +227,8 @@ typedef struct CXPLAT_RECV_DATA {
     //
     uint16_t Allocated : 1;          // Used for debugging. Set to FALSE on free.
     uint16_t QueuedOnConnection : 1; // Used for debugging.
-    uint16_t Reserved : 6;
+    uint16_t DatapathType : 2;       // CXPLAT_DATAPATH_TYPE
+    uint16_t Reserved : 4;
     uint16_t ReservedEx : 8;
```

Comment:
I'm not sure why we have separate Reserved fields. Let's just combine these into one.

Reply:
oops, I made the field?

Reply:
Reserved is used in `CxPlatDpRawRxEthernet` for packet parsing. Let me reduce ReservedEx size

Reply:
ReservedEx is also used. Reserved requires at least 3 bits, I can leave this as it is.

Reply:
Why can't we combine `Reserved` and `ReservedEx` into one `uint16_t Reserved : 12;`?

Reply:
`Reserved` and `ReservedEx` are used for two different things. If we combine them, it would be hard to consume them.

`Reserved` is socket type and `ReservedEx` is TCP header length. `ReservedEx` can be reduced to 6-bit but I don't think we need to touch them at all.